packages/expo-firebase-analytics/src/Analytics.ts

```diff
@@ -71,11 +71,16 @@ export async function setAnalyticsCollectionEnabled(isEnabled: boolean): Promise
   return await ExpoFirebaseAnalytics.setAnalyticsCollectionEnabled(isEnabled);
 }
 
+let hasWarnedAboutSetCurrentScreen = false;
 // @needsAudit
 /**
  * Sets the current screen name, which specifies the current visual context in your app. This helps
  * identify the areas in your app where users spend their time and how they interact with your app.
  *
+ * @deprecated FirebaseAnalytics.setCurrentScreen is deprecated.
+ * Use `logEvent('screen_view', { screen_name: 'MyScreen', ... })` instead.
+ * This API will be removed in SDK 45.
+
```

Comment:
```suggestion
 * @deprecated __Deprecated.__ Use [`logEvent('screen_view', { screen_name: 'MyScreen', ... })`](#logevent)
 * instead. This API will be removed in SDK 45.
 *
```
I will probably need to add some standardization for the comments from `@deprecated` annotation and apply the changes retrospectively. I think that the "Deprecated." message should be added automatically before the comment content. 😅 

Also I have added link to the recommended migration method.

Reply:
Awesome, thanks for that 😃